eslint.yaml

```diff
@@ -32,7 +32,7 @@ rules:
   guard-for-in: 2
   no-else-return: 2
   no-loop-func: 2
-  vars-on-top: 2
+  vars-on-top: 1
   no-debugger: 2
   no-cond-assign: 2
   no-console: 2
```

Comment:
Can we change this to a 1?  I'd like to see a warning about consoles, not an error.